Hadrons/Modules/MNPR/Bilinear.hpp

```diff
@@ -145,42 +145,49 @@ void TBilinear<FImpl>::execute(void)
     // Propagators
     auto  &qIn    = envGet(PropagatorField, par().qIn);
     auto  &qOut   = envGet(PropagatorField, par().qOut);
+    LatticeSpinColourMatrix qIn_phased(env().getGrid()), qOut_phased(env().getGrid());
```

Comment:
Please also use PropagatorField here. LatticeSpinColourMatrix is a hard-coded Grid type, part of which is e.g. the precision.

Reply:
Also, define as a temporary using envTmpLat()